fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/mv/AbstractMaterializedViewAggregateRule.java

```diff
@@ -60,15 +63,18 @@
  */
 public abstract class AbstractMaterializedViewAggregateRule extends AbstractMaterializedViewRule {
 
-    protected static final Map<Expression, Expression>
-            AGGREGATE_ROLL_UP_EQUIVALENT_FUNCTION_MAP = new HashMap<>();
+    // we only support roll up function which has only one argument currently
+    protected static final List<Pair<Expression, Expression>>
+            AGGREGATE_ROLL_UP_EQUIVALENT_FUNCTION_LIST = new ArrayList<>();
```

Comment:
`map<Expression, map<Expression, Expression>>` maybe better?

Reply:
Yeah you are right, I decide to use `MultiMap`, it's almost same to the `map<Expression, map<Expression, Expression>> `